src/Microsoft.Azure.EventHubs/Resources.Designer.cs

```diff
@@ -39,7 +39,7 @@ internal class Resources {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Azure.EventHubs.Resources", typeof(Resources).GetTypeInfo().Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Azure.EventHubs.Resources", typeof(Resources).Assembly);
```

Comment:
This change will probably break one of the builds. You can revert the change manually if needed.

Reply:
@serkantkaraca  Why is the reason this fail w/o `typeof(Resources).GetTypeInfo().Assembly`?

Reply:
I don't really remember the exact reason but I guess some assembly binding issue.

---
In reply to: [224279162](https://github.com/Azure/azure-event-hubs-dotnet/pull/324#discussion_r224279162) [](ancestors = 224279162)